specification/src/main/asciidoc/webprofile/Introduction.adoc

```diff
@@ -72,7 +72,7 @@ NOTE: Profile definitions can be quite terse, amounting to little more than a li
 Being the first profile of the Java(TM) EE 6 Platform to be defined, we expect the Web Profile specification to be used as a model for future profiles.
 It will also be seen as a starting point for understanding how the requirements defined in the Jakarta EE Platform specification apply to a profile that subsets the platform itself, a significant innovation in this version of the platform.
 (The case of a profile that is a superset of the platform is much easier to picture.)
-To help with this process, this section attempts to shed light on how one should go from the definition of the Web Profile to figuring out Jakarta EE 9 Web Profile, the exact set of requirements that apply to it, and consequently to any product that implements it.
+To help with this process, this section attempts to shed light on how one should go from the definition of the Web Profile to figuring out Jakarta EE Web Profile, the exact set of requirements that apply to it, and consequently to any product that implements it.
```

Comment:
Why did you drop '9' instead of changing to '9.1' ?  
It seems to be tautology.

Reply:
This section discusses "Jakarta EE" in general and this was the only reference to a specific version.  So, instead of constantly remembering to change this reference with each release, I thought I would simplify it and just remove the version reference.  But, now that I re-read this specific sentence, it does sound awkward.  Let me take another look...  Thanks!

Reply:
I simplified this sentence by removing the second reference to "Jakarta EE Web Profile".  I went back to the original wording in Java EE 8 Web Profile and it looks like we messed up when we translated to "Jakarta".  I like this simplified statement and it removes the specific reference to the version.  @kazumura, what do you think of the update?

Reply:
LGTM. Thanks.